### ISSUE TITLE
PyTorch教程：thorough-pytorch

### ISSUE THREAD

**Issue by SuperSupeng** [datawhalechina/DOPMC#19]
### 项目简介 
该项目希望大家能够掌握由浅入深地了解 PyTorch 的基本知识和内容，经过自己的动手实践加深操作的熟练度。同时通过项目实战，充分锻炼编程能力，掌握 PyTorch 进行深度学习的基本流程，提升解决实际问题的能力。
### 项目负责人
@LiJiaqi96 @NoFish-528 
### 项目地址
[thorough-pytorch](https://github.com/datawhalechina/thorough-pytorch)

**Comment by NoFish-528** [comment]
为了更好的完善教程，我们结合当前thorough-pytorch项目的进度，给出了以下task，希望有更多的同学能加入开源的贡献。主要有以下任务：
1. 补充深度学习相关知识（包括卷积，池化，全连接，激活函数，dropout，batch normalization，BP反向传播）（2人）
2. 补充计算机视觉领域、自然语言处理领域，语音处理相关简介（类比一篇论文的introduction，只需说明是什么，有哪些方法即可）（多人）
3. 介绍人工智能发展简史、DL,ML,AI三者之间的关系（1人）
4. 介绍评价相关指标（混淆矩阵， accuracy， precision， recall， F1 score）（1人）
5. Jupyter notebook或Jupyter Lab相关操作（1人）
6. 一些经典代码的解读（包括但不限于ResNet，GoogleNet，MobileNet，RCNN系列，SSD，YOLO系列，transformer系列，ViT，Detr，BERT，GPT系列，CLIP，GAN相关）（该任务意在介绍一些真正的代码工作是如何写的，建议解读的是PyTorch源码、官方实现的代码或者一些标准库代码包括但不限于timm等）（多人）

联系方式：
email: nzk020109@163.com 
wechat: ZhikangNiu_Wechat

@Sm1les @SuperSupeng

**Comment by NoFish-528** [comment]
为了更好的完善教程，我们结合当前thorough-pytorch项目的进度，给出了以下task，希望有更多的同学能加入开源的贡献。主要有以下任务：

1. 补充深度学习相关知识（包括卷积，池化，全连接，激活函数，dropout，batch normalization，BP反向传播）（2人）

2. 补充自然语言处理相关知识

3. 补充DP和DDP等分布式训练知识
3. 补充计算机视觉领域、自然语言处理领域，语音处理相关简介（类比一篇论文的introduction，只需说明是什么，有哪些方法即可）（多人）
4. 一些经典代码的解读（包括但不限于GoogleNet，MobileNet，RCNN系列，SSD，YOLO系列，transformer系列，ViT，Detr，BERT，GPT系列，CLIP，GAN相关）（该任务意在介绍一些真正的代码工作是如何写的，建议解读的是PyTorch源码、官方实现的代码或者一些标准库代码包括但不限于timm等）（多人）
联系方式：
email: [nzk020109@163.com](mailto:nzk020109@163.com)
wechat: ZhikangNiu_Wechat
@Sm1les @Truth-14

**Comment by NoFish-528** [comment]
thorough-pytorch 2.0教程更新计划
1. 补充更多自然语言处理的内容，介绍一些常见的任务和nlp的一个流程
2. 补充DP和DDP等分布式训练知识
3. 一些经典代码的解读（包括但不限于GoogleNet，MobileNet，RCNN系列，SSD，YOLO系列，transformer系列，ViT，Detr，BERT，GPT系列，CLIP，GAN相关）（该任务意在介绍一些真正的代码工作是如何写的，建议解读的是PyTorch源码、官方实现的代码或者一些标准库代码包括但不限于timm等）（多人）
联系方式：
email: [nzk020109@163.com](mailto:nzk020109@163.com)
wechat: ZhikangNiu_Wechat
@Sm1les @Truth-14
